draft-yasskin-http-origin-signed-responses.md

```diff
@@ -1568,6 +1648,12 @@ exchange argues for embedding a signature's lifetime into the signature.
 
 RFC EDITOR PLEASE DELETE THIS SECTION.
 
+draft-03
+
+* Define a CBOR structure to hold the certificate chain instead of re-using the
+  TLS1.3 message. Apparently TLS implementations don't expose their message
+  parsers enough to allow passing a message to a certificate verifier.
```

Comment:
(Also the TLS 1.3 message has different parsing rules than what you want anyway.)